bin/setup

```diff
@@ -1,4 +1,6 @@
 #!/usr/bin/env ruby
+
+require 'dotenv/load'
```

Comment:
This isn’t going to work very well (for first time runs) until *after* line 31 where we install gems.

Reply:
If we want to salvage using `bin/setup`, could we not rescue below the `bundle check` like:

```rb
begin 
  require 'dotenv/load'
rescue LoadError
end
```